tgui/packages/tgui/interfaces/PowerMonitor/Apc.tsx

```diff
@@ -8,7 +8,9 @@
 import { useBackend } from '../../backend';
 import { Box, Chart, LabeledList, Stack, Table, Tooltip } from '../../components';
 import { formatPower } from '../../format';
-import { PowerMonitorApcData, PowerMonitorApcItemData } from './type';
+import { SortDirection } from '../common/sorting/constant';
+import { Header } from '../common/sorting/Header';
+import { ApcTableHeaderColumns, ApcTableHeaderColumnSortState, numericCompare, PowerMonitorApcData, PowerMonitorApcItemData } from './type';
```

Comment:
`numericCompare` is not a type/interface/enum (enums are an edge case, but fit in the same bucket as types for logical grouping), should be in a `util` or similar file.

Reply:
moved sorting code to a common/comparators.ts file since this can be reused, specifically in #18246.